resources/indexer.py

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
Consider using a CVMFS metaproject with shebang so it auto-loads the env.
http://software.icecube.wisc.edu/documentation/info/cvmfs.html#self-contained-icetray-scripts

Reply:
What's the metaproject in this case?

Reply:
Either the latest release, or we maybe could use combo stable.